src/apps/backend/modules/account/account-service.ts

```diff
@@ -106,4 +106,8 @@ export default class AccountService {
 
     return AccountWriter.deleteAccountById(accountId);
   }
+
+  public static async getAllAccounts(params: string): Promise<Account[]> {
```

Comment:
`params: string`: The name of the arg is ambiguous and not sure what it `params` refer to